post_install.yml

```diff
@@ -5,13 +5,12 @@
     secure: false
     ASSISTED_INSTALLER_BASE_URL: "{{ secure | ternary('https', 'http') }}://{{ hostvars['assisted_installer']['host'] }}:{{ hostvars['assisted_installer']['port'] }}/api/assisted-install/v1"
     URL_ASSISTED_INSTALLER_CLUSTER: "{{ ASSISTED_INSTALLER_BASE_URL }}/clusters/{{ cluster_id }}"
-    kube_filename: "{{ kubeconfig_dest_filename | default('kubeconfig') }}"
+    kube_filename: "{{ kubeconfig_dest_filename | default(cluster_name + '-kubeconfig') }}"
     dest_dir: "{{ kubeconfig_dest_dir | default(ansible_env.HOME) }}"
     kubeconfig_path: "{{ dest_dir }}/{{ kube_filename }}"
-
+    kubeadmin_vault_name: "{{ cluster_name }}-kubeadmin.vault.yml"
```

Comment:
We have an inventory var for the kubeconfig filename (`kubeconfig_dest_filename`), but we don't have one for the kubeadmin vault -- its filename is hardcoded in `post_install.yml`.

It's a little bit inconsistent, but I won't be pushing back if you feel like adding a config var to the inventory is just not worth it.


```yaml
# The inventory's kubeconfig section could look like this
kubeconfig_dest_dir: /home/redhat/
kubeconfig_dest_filename: "{{ cluster_name }}-kubeconfig"
kubeadmin_dest_filename: "{{ cluster_name }}-kubeadmin.vault.yml"
```

Reply:
I think that makes sense We already did it for kubeconfig and it makes it easier for people to find the values